src/pages/ReportDetailsPage.tsx

```diff
@@ -79,8 +79,8 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
     const {translate} = useLocalize();
     const {isOffline} = useNetwork();
     const styles = useThemeStyles();
-    const [parentReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID ?? ''}`);
-    const [sortedAllReportActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID ?? ''}`, {
+    const [parentReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID ?? '-1'}`);
+    const [sortedAllReportActions = []] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID ?? '-1'}`, {
```

Comment:
Is this change still required to fix crash - onyx version bump itself doesn't fix undefined issue?

Reply:
Confirmed it now catches incorrect typings 👍 

![image](https://github.com/Expensify/App/assets/108292595/38a6474f-9bf0-49a1-9e60-6a1c8fd91d74)